projects/demo/src/modules/components/input-password/examples/2/index.html

```diff
@@ -0,0 +1,9 @@
+<form class="b-form" [formGroup]="testForm">
+    <tui-input-password
+        formControlName="testValue"
+        tuiHintDirection="right"
+        tuiHintMode="onDark"
```

Comment:
Default hint also follows tuiMode directive so if the input is on dark background it automatically turns white. Maybe this extra hint should also do that?

Reply:
Implemented this logic